src/Pos/Update/Poll/Pure.hs

```diff
@@ -0,0 +1,70 @@
+-- | Pure Poll
+
+module Pos.Update.Poll.Pure
+       ( PurePoll (..)
+       ) where
+
+import           Control.Lens               (to, (%=), (.=))
+import qualified Data.HashMap.Strict        as HM
+import qualified Data.HashSet               as HS
+import           System.Wlog                (CanLog, HasLoggerName, NamedPureLogger)
+import           Universum                  hiding (toList)
```

Comment:
Also I personally don't like `toList` name for `MapModifier` and how functions are named. Naming is inconsistent with all other _Haskell_. `toList` functions and similar are for `Foldable` type class and should have clear semantic with similar types.